packages/manager/apps/web/client/app/hosting/cdn/shared/cors/routing.js

```diff
@@ -0,0 +1,27 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.hosting.dashboard.cdn.shared.cors', {
+    url: '',
```

Comment:
No url ?

Reply:
In fact it would be weird to directly access cors rules by browsing an url as the path to edit cors is :
Select a hosting > go to multisite section > select the CDN to edit > add cors rules